net/vallumd/Makefile

```diff
@@ -0,0 +1,54 @@
+#
+# Copyright (C) 2017 Stijn Tintel <stijn@linux-ipv6.be>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=vallumd
+PKG_VERSION:=0.1.0
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL=https://github.com/stintel/vallumd/archive/$(PKG_VERSION)/
+PKG_MD5SUM:=81f6a721a2f2b5fefcec9217bafb121d6a969440a9165da8d5a3084704743e5c
+
+PKG_LICENSE:=GPL-3.0
+PKG_LICENSE_FILES:=COPYING
+
+PKG_MAINTAINER=Stijn Tintel <stijn@linux-ipv6.be>
+
+CMAKE_INSTALL:=1
```

Comment:
There is no use for this CMAKE_INSTALL variable?  PKG_INSTALL should be what you want and it will be turned on by default in cmake.mk